packages/docs/docs/docs/deploying.md

```diff
@@ -56,14 +56,20 @@ by now, inside the `my-project` directory you should have a `package.json` file
 a `truffle-config.js` file (created by the `zos` CLI for Truffle), and a
 `zos.json` file (created by `zos` for ZeppelinOS).
 
+Note: if you haven't installed Truffle, run the following command:
+
+```console
+npm install -g truffle
```

Comment:
```suggestion
npm install truffle
```

I wouldn't suggest globall install until https://github.com/zeppelinos/zos/issues/579 is fixed